portal/account/account.lib.php

```diff
@@ -0,0 +1,205 @@
+<?php
+/**
+ * Ajax Library for Register
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2017 Jerry Padgett <sjpadgett@gmail.com>
+ * @license https://www.gnu.org/licenses/agpl-3.0.en.html GNU Affero General Public License 3
+ */
+/* Library functions for register*/
+
+function notifyAdmin($pid, $provider)
+{
+
+    $note = xlt("New patient registration received from patient portal. Reminder to check for possible new appointment");
+    $title = xlt("New Patient");
```

Comment:
TODO for me:
remove html escaping above since this is going to database (not html)

Reply:
Oops, I'll fix with password reset.

Reply:
Don't worry about these little things since likely gonna be a couple more of these on my last code review/sweep (I am happy to make these minor changes before bringing this awesome feature into the codebase). Just let me know when password reset is in and I'll resume the last code review/sweep/commit :)